jest.config.ts

```diff
@@ -25,10 +25,6 @@ const config: Config.InitialOptions = {
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
   testEnvironment: 'jsdom',
   setupFilesAfterEnv: ['<rootDir>/src/testUtils/setupTests.ts'],
-
-  transformIgnorePatterns: [
```

Comment:
We should not compile whole node_modules. Let's figure out where the issue is coming from and compile just that dependency

Reply:
IIUC the default behavior of jest is to not compile node_modules
https://jestjs.io/docs/configuration#transformignorepatterns-arraystring
I copied this field from mce, it contains patterns that don't exist in our code and turns out this field is unnecessary altogether